src/datepicker/datepicker.spec.ts

```diff
@@ -541,6 +572,143 @@ describe('ngb-datepicker', () => {
       expect(getYearSelect(fixture.nativeElement).value).toBe(`${today.getFullYear()}`);
     });
 
+    it('should correctly navigate with keyboard', () => {
```

Comment:
Think of it of as a big integration test :) 
key-mapping service has unit tests of its own, but yes, we should split this one